embedded/devices/include/rms.h

```diff
@@ -0,0 +1,42 @@
+#ifndef _RMS__H__
+#define _RMS__H__
+
+#include "can.h"
+
+#define RMS_EEPROM_SEND_ID 0xC1
+#define RMS_HB_ID 0xC0
+#define RMS_CLR_FAULTS_ID 0xC1
+#define RMS_INV_DIS_ID 0xC0
+#define RMS_INV_EN_ID 0xC0
+#define RMS_INV_FW_10_ID 0xC0
```

Comment:
It seems kinda silly to define 3 different macros that all mean make the motor go forward at 0,10,20,30 NM.

The datasheet defines this as the COMMAND_MESSAGE id. Lets call it that instead